docs/guide/writing/support-team.rst

```diff
@@ -30,3 +37,34 @@ My advice: don’t be discouraged or alarmed if you get questions you can’t an
 
 So, how can you deal with this safety net being removed? Firstly, recognize that it’s fine not to know something – never give an answer you’re not 100% sure about just for the sake of saving face. If you don’t know, it’s always a good idea to ask another agent for their input, or escalate the ticket to the next support level for support.
+
+Tips for tech writers who want to work in their own role within support
+-----------------------------------------------------------------------
+
+If you work for a large-scale SaaS business, being a tech writer involves keeping up with gaps in the documentation, maintaining versions, coordinating release notes, and so on.
+
+Often this means that you have a huge backlog of tasks to juggle, and you cannot take on another role, like supporting client. However, this does not mean that you should lose your contact with this essential part of the business. Working as a technical writer within a support team is a great way to keep up with customer issues while maintain the documentation. All you need is a solid plan to make it happen.
+
+Consider these tips: 
+
+* Get the team on board
+* Make communication simple
+* Collaborate
+
+Get the team on board
+~~~~~~~~~~~~~~~~~~~~~
+
+It is always important to remember that when you work as a separate role within a team, it does not in fact make you apart from the team. Even though you write, and they support, you are still one team who are essential to each other. The better the documentation is, the less redundancies there are in tickets and calls coming in. This means that support will get less tickets, yes, but those tickets will contain challenging issues that really require their time and skills. This is a rewarding, symbiotic relationship. When they have meetings on issues, you attend; when you have doc issues, you communicate those with them. You can’t be a team unless you understand each other and the unique issues each of you face. 
+
+Make communication simple
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you are writing the documentation full-time, you have no time to answer calls or respond to tickets. You are relying on your team to let you know which issues are coming in often enough to create a doc page or FAQ question. However, they are busy too. Anyone with exposure to support work will know that it is tough job that requires helping as many customers as you can in a single day while maintaining the highest satisfaction rating possible. So, they need to be able to send you their issues quickly and efficiently. There are several ways to do this, creating a dedicated email account for doc tasks where they can send issues, creating a Slack command that assigns you the task automatically, having a team Trello board, and so on. Find the solution that works best for your team. And don’t despair – if one solution does not work, move on to the next.
```

Comment:
Rephrasing bits and pieces as follows; please comb through each phrase:

"When you are writing documentation full-time, you have little to no time to answer calls or respond to tickets. You are relying on your team to let you know about any issues that resurface enough to create a doc page or FAQ. However, they are busy too. Anyone with exposure to support work will know that it is a tough job that requires helping as many customers as you can in a single day while maintaining the highest satisfaction rating possible. The support team needs to be able to send you their issues quickly and efficiently.

There are several ways to do this:

* Create a dedicated email account for doc tasks where they can send issues.
* Create a Slack command that assigns you the task automatically.
* Use a team Trello board.

Find a solution that works best for your team, and don’t despair. If one solution does not work, move on to the next."